doc/pages/user/015-astarte_dashboard.md

```diff
@@ -0,0 +1,394 @@
+
+## Introduction
+
+The Astarte Dashboard is a Single Page Application to get an overview of and manage your Realm on
+any browser. The Dashboard is designed to be a quick and easy way to give you immediate feedback on
+your work and as a quick and intuitive way to configure your realm. It is not meant to be operated
+by end-users, rather by infrastructure maintainers, owners who need information on the system
+status, and those working on projects based on Astarte. It is shipped by default with the Astarte
+standard distribution.
+
+The Dashboard is a graphical client for Astarte APIs; it shares similar features with the CLI client
+[astartectl](https://github.com/astarte-platform/astartectl), the command-line utility to manage
+Astarte.
+
+The Dashboard helps you manage:
+
+- [Triggers](015-astarte_dashboard.html#triggers)
+- [Interfaces](015-astarte_dashboard.html#interfaces)
+- [Devices](015-astarte_dashboard.html#devices)
+- [Groups](015-astarte_dashboard.html#groups)
+- [Realm Settings](015-astarte_dashboard.html#realm-settings)
+
+In case your Astarte distribution comes packaged with the
+[Flow framework](https://docs.astarte-platform.org/flow/snapshot), the Dashboard is probably
+configured to manage Flow resources as well.
+
+- [Blocks](015-astarte_dashboard.html#blocks)
+- [Pipelines](015-astarte_dashboard.html#pipelines)
+- [Flows](015-astarte_dashboard.html#flows)
+
+Please note that the Flow framework is not available for use in a docker-compose environment since
+it relies on Kubernetes APIs to operate.
+
+### How to access it
+
+Depending on how you are using Astarte, here is where you can find the Dashboard:
+
+- Docker-compose: if you are using a local instance of Astarte via `docker-compose`, you will find
+  it by pointing your browser to the default address `http://localhost:4040/`. To login, fill in the
+  name of your realm and a valid JWT token: if you possess the realm private key, as in the
+  [Astarte in 5 minutes](010-astarte_in_5_minutes.html) guide, you can generate the token with the
+  command `astartectl utils gen-jwt all-realm-apis -k <private_key>`.
+- Astarte Cloud: if you are using our managed Astarte option, you can hop onto our
+  [Console](https://console.astarte.cloud/realms), find or create your realm and click the Dashboard
+  button.
+- Own Kubernetes cluster: in this case, if the Dashboard is enabled, it can be usually found at
+  `dashboard.<base Astarte URL>`. Refer to your system administrator for more details.
+
+## Main overview
+
+Upon successful login, the main screen is the home page that provides an overview of the realm
+status and resources.
+
+![Astarte Dashboard with a realm overview](assets/astarte-dashboard-realm-overview.png)
+
+### API Status
+
+The API Status gives you general information about the status of services:
+
+- **Realm Management** is an administrator-like API for configuring a Realm. It is used for managing
```

Comment:
Realm Management does not handle devices. It only manages the installation/retrieval of interfaces and triggers.

Reply:
Oh, I copied that line from https://docs.astarte-platform.org/1.0/020-components.html#realm-management
I guess we should change that too.

Reply:
Fixed them both.